metadata_test.go

```diff
@@ -0,0 +1,264 @@
+
+			Expect(file.Config.User).To(Equal("1000:1000"))
+
+			Expect(file.Config.Env).NotTo(ContainElements(
+				"CNB_USER_ID=1000",
+				"CNB_GROUP_ID=1000",
+				"CNB_STACK_ID=io.paketo.stacks.tiny",
+			))
+
+			Expect(image).To(SatisfyAll(
+				HaveFileWithContent("/etc/group", ContainSubstring("cnb:x:1000:")),
+				HaveFileWithContent("/etc/passwd", ContainSubstring("cnb:x:1000:1000::/home/cnb:/sbin/nologin")),
+				HaveDirectory("/home/cnb"),
+			))
+
+			Expect(image).To(SatisfyAll(
+				HaveFile("/usr/share/doc/ca-certificates/copyright"),
+				HaveFile("/etc/ssl/certs/ca-certificates.crt"),
+				HaveDirectory("/root"),
+				HaveDirectory("/tmp"),
+				HaveFile("/etc/services"),
+				HaveFile("/etc/nsswitch.conf"),
+			))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/base-files", SatisfyAll(
+				ContainSubstring("Package: base-files"),
+				MatchRegexp(`Version: [0-9\.]+ubuntu[0-9\.]+`),
+				ContainSubstring("Architecture: amd64"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/ca-certificates", SatisfyAll(
+				ContainSubstring("Package: ca-certificates"),
+				MatchRegexp("Version: [0-9]+"),
+				ContainSubstring("Architecture: all"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/libc6", SatisfyAll(
+				ContainSubstring("Package: libc6"),
+				MatchRegexp("Version: [0-9\\.\\-]+ubuntu[0-9\\.]+"),
+				ContainSubstring("Architecture: amd64"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/libssl1.1", SatisfyAll(
+				ContainSubstring("Package: libssl1.1"),
+				MatchRegexp("Version: [0-9\\.\\-]+ubuntu[0-9\\.]+"),
+				ContainSubstring("Architecture: amd64"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/netbase", SatisfyAll(
+				ContainSubstring("Package: netbase"),
+				MatchRegexp("Version: [0-9\\.]+"),
+				ContainSubstring("Architecture: all"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/openssl", SatisfyAll(
+				ContainSubstring("Package: openssl"),
+				MatchRegexp("Version: [0-9\\.\\-]+ubuntu[0-9\\.]+"),
+				ContainSubstring("Architecture: amd64"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/tzdata", SatisfyAll(
+				ContainSubstring("Package: tzdata"),
+				MatchRegexp("Version: [a-z0-9\\.\\-]+ubuntu[0-9\\.]+"),
+				ContainSubstring("Architecture: all"),
+			)))
+
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status.d/zlib1g", SatisfyAll(
+				ContainSubstring("Package: zlib1g"),
+				MatchRegexp("Version: [a-z0-9\\.\\-\\:]+ubuntu[0-9\\.]+"),
+				ContainSubstring("Architecture: amd64"),
+			)))
+
+			Expect(image).NotTo(HaveFile("/usr/share/ca-certificates"))
+
+			Expect(image).To(HaveFileWithContent("/etc/os-release", SatisfyAll(
+				// TODO: This is different from what we currently ship. Are we ok
+				// with changing?
+				ContainSubstring(`PRETTY_NAME="Paketo Buildpacks Tiny Bionic"`),
```

Comment:
Question for @paketo-buildpacks/stacks-maintainers  – I've updated the contents of the `os-release` file to be a bit more up-to-date compared to what we [ship in the current Tiny stack](https://github.com/paketo-buildpacks/stacks/blob/53374cadbf79a64f2290708337aaa0174f24fc14/tiny/dockerfile/run/files/os-release).

Reply:
I think this is OK. I'm extremely surprised to see the Cloud Foundry references in the current Tiny stack and I'd like to get away from that. I can't imagine that is breaking information either.